app/views/works/form.html.erb

```diff
@@ -53,6 +53,19 @@
         <%= render Elements::Forms::TextareaFieldComponent.new(form:, field_name: :title, label: t('works.edit.fields.title.label'), tooltip: t('works.edit.fields.title.tooltip_html'),
                                                                help_text: t('works.edit.fields.title.help_text'), width: 500, rows: 1, container_classes: 'pane-section') %>
         <%= render NestedComponentPresenter.for(form:, field_name: :contact_emails, model_class: ContactEmailForm, form_component: Works::Edit::ContactEmailsComponent, hidden_label: true, bordered: false, single_field: true, fieldset_classes: 'pane-section') %>
+
+        <% if @work_form.deposits_contact_email %>
+          <%= form.hidden_field :deposits_contact_email %>
+
+          <div class="pane-section">
+            <p>
+              Contact email provided by collection manager <%= render Elements::TooltipComponent.new(tooltip: t('works.edit.fields.deposits_contact_email.tooltip_html')) %>
+            </p>
+            <p>
+              <%= @work_form.deposits_contact_email %>
+            </p>
+          </div>
```

Comment:
This is the work form UI for collection contact emails: keep the field in the form, but hidden, and let the user know their collection manager has set a contact email for their work that they cannot edit.